frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -366,11 +367,18 @@ func (h *startCVDHandler) Launch(instanceNumber uint32, artifactsDir, homeDir st
 	return nil
 }
 
-func createDir(dir string, failIfExist bool) error {
-	// TODO(b/238431258) Use `errors.Is(err, fs.ErrExist)` instead of `os.IsExist(err)`
-	// once b/236976427 is addressed.
+func createDirFailIfExist(dir string) error {
+	err := os.Mkdir(dir, 0774)
+	if err != nil {
+		return err
+	}
+	// Mkdir set the permission bits (before umask)
+	return os.Chmod(dir, 0774)
+}
+
+func createDir(dir string) error {
```

Comment:
nit: this function could be implemented as:
```
err := createDirFailIfExists(dir)
if os.IsExist(err) {
    return nil
} else {
    return err
}
```

Reply:
good call! done